src/collection.js

```diff
@@ -96,6 +96,13 @@ function markSynced(record) {
   return markStatus(record, "synced");
 }
 
+function listStatuses(collection, statuses, options={includeDeleted: true}) {
+  return Promise.all(statuses.map(status => {
+    return collection.list({order: "", filters: {_status: status}}, options);
```

Comment:
Now I think a little more about it, we could probably directly use the adapter api here. Will experiment.